src/utils/enrichers.ts

```diff
@@ -75,3 +80,16 @@ export const fillSingleProject = async (
     projectId: data.projects[0].id,
   };
 };
+
+export const fillSingleOrg = async (
+  props: CommonProps & Partial<Pick<OrgScopeProps, 'orgId'>>,
```

Comment:
I don't fully understand what you are saying. I was just following the example of `fillSingleProject` just above.

Reply:
well, OrgScopeProps is defined as CommonProps + the extra orgId field
this type is CommonProps, plus just the orgId feild picked from OrgScopeProps- which is the same thing, right?

Reply:
if theres a documentation purpose then it would make sense, but I'm just pointing out that the type signature raises more questions than it answers :sweat_smile:

Reply:
It seems to me that it matters for the *return type* of the function (which is inferred implicitly here).
If we do not mention `CommonProps` in the argument, then the inferred return type will also not contain `CommonProps`. Is that what we want?

Reply:
I'm wondering if it would work the same with `props: OrgScopeProps`- if not then ignore this, I just assume typescript should be smart enough to figure this out

Reply:
NIT

Reply:
I tried removing it and I encountered the following error:
```
src/utils/enrichers.ts:90:32 - error TS2339: Property 'apiClient' does not exist on type 'Partial<Pick<OrgScopeProps, "orgId">>'.

90   const { data } = await props.apiClient.getCurrentUserOrganizations();
```

I think that settles it?

Reply:
ah no, not `Partial<Pick....`, just `props: OrgScopeProps`. AFAIU partial pick says "give me these specific parts of this type"

Reply:
Ok I have simplified (and also `fillSingleProject` above) and it seems to be ok.